tensorflow/core/kernels/quantization_utils.h

```diff
@@ -268,7 +268,7 @@ inline void RequantizeManyInNewRangeReference(const qint32* input, int64 count,
   // that could be easily adapted for a SIMD implementation. It should also be
   // possible to perform all the calculations in 32-bit rather than 64, but
   // that's not been implemented yet.
-  for (size_t index = 0; index < count; ++index) {
+  for (size_t index = 0; static_cast<tensorflow::int64>(index) < count; ++index) {
```

Comment:
Same here

Reply:
I'll apply the new styling to all of the changes affecting for loops.

Reply:
Actually, can you clarify @mihaimaruseac ? I am not sure of your request.

Reply:
Was suggesting to replace all `for (type var = 0; static_cast<other_type>(var) < limit_expr; ...)` to `for (type var = 0, type limit = limit_expr; var < limit; ...)` if `limit_expr` is an expression. Otherwise, if `limit_expr` is just a variable, we can change type of `var` or `limit_expr` variable at declaration so that they match.